### PR TITLE
android - example of new UriAction subclassing

### PR DESCRIPTION
**Pull request by radixdev**
# Pull Request/Issue Resolution
 docs for https://github.com/Appboy/android-sdk/pull/2847

# release when?
should be merged after https://github.com/Appboy/android-sdk/pull/2847 goes live

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @KellieHawks as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

**Comment by KellieHawks** [comment]
@radixdev Please let me know when Appboy/android-sdk#2847 is live. I will merge soon after that. Thank you!

File: _includes/archive/android/universal_deep_link_delegate.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    private val TAG = AppboyLogger.getAppboyLogTag(CustomAppboyNavigator::class.java)
+  }
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+[udl-1]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/ui/actions/UriAction.html
+[udl-2]: https://github.com/Appboy/appboy-android-sdk/blob/master/CHANGELOG.md#200
```
[the remark this answers is not in the document]
Comment (radixdev) [reply _includes/archive/android/universal_deep_link_delegate.md]: fixed

**Comment by radixdev** [comment]
@KellieHawks still unreleased as of yet. Apologies if tagging you for review was premature

**Comment by KellieHawks** [comment]
> @KellieHawks still unreleased as of yet. Apologies if tagging you for review was premature

@radixdev No worries, thanks for the update!

**Comment by radixdev** [comment]
@KellieHawks this is now live

**Pull request merged**